bin/oneoff/data_fix/fix_bad_school_info_in_users

```diff
@@ -64,8 +75,10 @@ CSV.open(csv_file, 'w') do |csv|
 
     users.each do |user|
       csv << [user.id.to_s, user.school_info_id.to_s]
-      if ACTUALLY_UPDATE
-        user.update!(school_info_id: nil)
+      if options[:actually_update]
+        user.last_seen_school_info_interstitial = DateTime.now + options[:interstitial_days]
+        user.school_info_id = nil
+        user.save!(validate: false)
```

Comment:
Why skip validation? I'm guessing that's for the same reason as [this](https://github.com/code-dot-org/code-dot-org/blob/8d961a3524e323e867e2d42e31a16ced3a8f4691/lib/cdo/school_info_interstitial_helper.rb#L22-L25), in which case please add a small comment. 

As an aside, we really should try to fix our users table so they all pass validation at some point :(